airbyte-integrations/connectors/destination-timeplus/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: f70a8ece-351e-4790-b37b-cb790bcd6d54
-  dockerImageTag: 0.1.17
+  dockerImageTag: 0.1.18
```

Comment:
@jovezhong the reason why Timeplus is not updated in Airbyte Cloud is the `registryOverrides` is set to false. Can you update to true?

Reply:
sure, I will update the PR shortly.

Reply:
I have updated the 2 enabled values